lib/go/test/market_test.go

```diff
@@ -62,9 +62,25 @@ func TestMarketDeployment(t *testing.T) {
 	_, err = b.CommitBlock()
 	assert.NoError(t, err)
 
+	// Should be able to deploy the TopShotLocking contract
+	// as a new account with no keys.
+	topShotLockingCode, _ := DownloadFile(TopShotLockingContractsBaseURL + TopShotLockingContractsInterfaceFile)
```

Comment:
Instead of `DownloadFile`, you should be able to add a new method to the `contracts` package (`GenerateTopShotLockingContract`) and use that to get the contract code. We usually try to avoid downloading directly.